person_capture/gui_app.py

```
@@ -5389,14 +5443,223 @@ def _resolve_ffmpeg_bin(self) -> Optional[str]:
+        return max(1, fw), max(1, fh)
+
+    @staticmethod
+    def _capture_frame_pts_sec(cap, frame_idx: int, fps: float) -> Optional[float]:
+        """Return frame timestamp seconds, preferring capture-reported time over frame_idx/fps."""
+        try:
+            if cap is not None and hasattr(cap, "get"):
+                pos_msec = float(cap.get(cv2.CAP_PROP_POS_MSEC) or 0.0)
+                if math.isfinite(pos_msec) and pos_msec >= 0.0:
+                    return pos_msec / 1000.0
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)